src/site/README.md

```diff
@@ -0,0 +1,309 @@
+
+<details><summary>1. Determine the value of the "entityBundle" belonging to the new type of page
+</summary>
+
+This new type of page will have a unique value for its top-level `entityBundle`. As described in previous sections, this value will be used to form the relationship between the CMS data structure and the template. Thus, determining this value and using it to learn about your data structure is the first part of getting started.
+
+The simplest way of getting your page's `entityBundle` is by leveraging the usual avenues on Slack to connect with the CMS team (primarily the `#cms-support` channel) or by simply asking your supporting CMS engineer what the value is. This value can be considered the "hand-off" to the front-end team so that templating can begin.
+
+</details>
+
+<details>
+
+<summary>2. Write the GraphQL query</summary>
+
+The CMS provides a tool called the [GraphQL Explorer](https://prod.cms.va.gov/graphql/explorer) to help write and debug GraphQL queries. You may be familiar with this tool from other GraphQL servers. You can leverage the value of the `entityBundle` determined in the former step to get started writing your query. If you are not familiar with GraphQL queries, the GraphQL website provides fantastic [documentation](https://graphql.org/learn/queries/) we recommend reading.
+
+A GraphQL query for retrieving pages will consist of a `nodeQuery` with a `filter` applied to narrow down results to only nodes belonging to a certain `entityBundle`. Consider this GraphQL snippet used for getting all pages of type `landing_page`. In this example, `landing_page` is the value of the `entityBundle`.
+
+```graphql
+{
+  nodeQuery(limit: 1, filter: {conditions: [{field: "type", value: ["landing_page"]}]}) {
+    entities {
+      ... on NodeLandingPage {
+        entityId
+      	entityBundle
+        title
+        fieldIntroText
+      }
+    }
+  }
+}
+```
+
+The response JSON for this GraphQL query will contain a single instance of the page-type `landing_page`. Here is the response JSON as of writing.
+
+```json
+{
+  "data": {
+    "nodeQuery": {
+      "entities": [
+        {
+          "entityId": "79",
+          "entityBundle": "landing_page",
+          "title": "VA records",
+          "fieldIntroText": "Access your VA records and documents online to more easily manage your benefits."
+        }
+      ]
+    }
+  }
+}
+```
+
+Your complete query will be restructured so that your JS module exports a GraphQL fragment for use in the preview server as well as as a standalone `nodeQuery` for use in the content build. It should also contain an additional `filter` - a boolean field called `status` that is used to toggle only draft vs. published content. Here is a complete example of what your module may look like. _Note - Hopefully, soon the preview server will be updated to use the same query as the content build. This doc wil be updated once that happens._
```

Comment:
Maybe add a link on the word "fragment" to https://graphql.org/learn/queries/#fragments just to make it super clear that we're referring to GraphQL fragments here and not some VA- or Drupal- specific thing?